packages/blobs/src/main.test.ts

```diff
@@ -1990,6 +2057,56 @@ describe(`getStore`, () => {
       'Netlify Blobs could not find a `fetch` client in the global scope. You can either update your runtime to a version that includes `fetch` (like Node.js 18.0.0 or above), or you can supply your own implementation using the `fetch` property.',
     )
   })
+
+  test('Uses explicit siteID and token from options instead of context values', async () => {
```

Comment:
This is not really related to this PR, but there was no test for this behaviour.